sim/pokemon.ts

```diff
@@ -602,33 +602,21 @@ export class Pokemon {
 	}
 
 	/** Get targets for Dragon Darts */
-	getSmartTargets(target: Pokemon, move: Move) {
+	getSmartTargets(target: Pokemon, move: ActiveMove) {
```

Comment:
Do we need this function?

Reply:
This seems short enough that we might want to inline it in `getMoveTargets`... but then again, maybe not.

Reply:
I think we might still need it to compute the initial list of targets depending on Ally Switch etc.

Reply:
Understandable. Let's keep this in for now.